### PR TITLE
Add SAP Connector documentation

### PR DESCRIPTION
**Pull request by luispinedamx**

### CODE REVIEW

File: src/docs/user/references/DataConnectors.adoc [hunk 1]
```diff
@@ -291,3 +291,83 @@ You have two options for providing your login credentials:
...
+  println("System info:")
+  // loop on structure to get the info
+  for(JCoField field : structure) {
+    println(field.getName() + " : " + field.getString())
+  }
+}
+----
+
+Further information and examples using the SAP JCO library are available in the link:https://help.hana.ondemand.com/javadoc/index.html[SAP Cloud Platform SDK
+Documentation] and the SAP Java Connector documentation accessible through link:https://websmp201.sap-ag.de/[SAP Service Marketplace].
```
Comment (nebilbenmabrouk) [inline src/docs/user/references/DataConnectors.adoc]: the SAP Java Connector documentation **is** accessible through **the** link

File: src/docs/user/references/DataConnectors.adoc [hunk 2]
```diff
@@ -291,3 +291,83 @@ You have two options for providing your login credentials:
...
+| `JCO_LANG`
+| Preferred language.
+| EN
+| `JCO_SAPROUTER`
+| _(Optional)_ An SAP Router string, in case you are connecting from an external
+               network. It is important that you keep the /H/ tags.
+| /H/192.168.0.1/H/
+|===
+
+Once the credentials are set, the workflow can be executed. If the connection is successful, the connection attributes will be listed as output. *NB.* Proactive's SAP connector provides libraries to connect to an SAP server from a 64-bit OS (Windows, Linux, MacOS); libraries for other OS can be obtained through SAP Marketplace.
```
Comment (nebilbenmabrouk) [inline src/docs/user/references/DataConnectors.adoc]: to connect to **a** SAP server

Comment (luispinedamx) [reply src/docs/user/references/DataConnectors.adoc]: I had some doubt but I think "an SAP server" is correct, as it's read "an ess-a-pee server"
http://blog.apastyle.org/apastyle/2012/04/using-a-or-an-with-acronyms-and-abbreviations.html

File: src/docs/user/references/DataConnectors.adoc (hunk 1 above)
Comment (luispinedamx) [reply src/docs/user/references/DataConnectors.adoc]: The word "link" itself is not visible. When rendered, the sentence reads "and the SAP Java Connector documentation, accessible through SAP Service Marketplace" (I might be missing a comma ;) )

File: src/docs/user/references/DataConnectors.adoc [hunk 3]
```diff
@@ -291,3 +291,83 @@ You have two options for providing your login credentials:
...
 * Once downloaded, the output file will be stored in your *User Space* (and _not_ in the Global Space).
 ====
+
+=== ERP
+
+Proactive's ERP connectors enable communication and data exchange with popular ERP software providers.
+
+==== SAP ECC
+
+This connector allows you to interact with an SAP ECC system (not S/4HANA) by means of remote function calls (RFC).
```
Comment (amouhoub) [inline src/docs/user/references/DataConnectors.adoc]: maybe:
Remote Function Calls (RFC).
(with capitals)

File: src/docs/user/references/DataConnectors.adoc [hunk 4]
```diff
@@ -291,3 +291,83 @@ You have two options for providing your login credentials:
...
+
+*Variables:*
+
+.SAP ECC Connector variables
+[cols="2,5,2,2,2,2"]
+|===
+| *Variable name* | *Description* | *Level* | *Required?* | *Type*  | *Default/Examples*
+| `JCO_ FUNCTION`
+| the name of a remote-enabled function module on your SAP System. You can verify the available functions in your ECC transaction SE37.
```
Comment (amouhoub) [inline src/docs/user/references/DataConnectors.adoc]: The name of ...
(capital T)

File: src/docs/user/references/DataConnectors.adoc [hunk 5]
```diff
@@ -291,3 +291,83 @@ You have two options for providing your login credentials:
...
+| `JCO_SAPROUTER`
+| _(Optional)_ An SAP Router string, in case you are connecting from an external
+               network. It is important that you keep the /H/ tags.
+| /H/192.168.0.1/H/
+|===
+
+Once the credentials are set, the workflow can be executed. If the connection is successful, the connection attributes will be listed as output. *NB.* Proactive's SAP connector provides libraries to connect to an SAP server from a 64-bit OS (Windows, Linux, MacOS); libraries for other OS can be obtained through SAP Marketplace.
+
+.Querying for a function
+The SAP connector will search for the function module provided as the `JCO_FUNCTION` variable. If the function module exists and is remote-enable, the script will display the function's parameters (import, export, tables) right after the Connection attributes.
```
Comment (amouhoub) [inline src/docs/user/references/DataConnectors.adoc]: If the function module exists and is remote-enable
maybe:
is remote-enabled (not sure)
or:
... and if remote-enable is activated (or set to true)

Comment (luispinedamx) [reply src/docs/user/references/DataConnectors.adoc]: nice catch! the first one is correct

**Pull request merged**